src/schema/types.ts

```diff
@@ -147,3 +148,24 @@ export type DataTrackedAttrs = {
   userID: string
   createdAt: number
 }
+
+export type SectionCategory = {
+  _id: string
+  name: string
+  synonyms: string[]
+  isEditable: boolean
+  isDisabled?: boolean
+  titles?: string[]
+  groupIDs?: string[]
```

Comment:
The idea with `groupIDs` never materialized, so I think we should revisit it. Maybe we can get rid of it completely, and update the template to have a separate list for body section categories and backmatter section categories?

Reply:
So, you mean we'd have 4 lists? abstracts, body, backmatter and other sections? And in the [body-editor](https://github.com/Atypon-OpenSource/manuscripts-body-editor/blob/6d24b2031be40663f75b20d2f03d089ca3ac5582/src/plugins/section-category/section-category-utils.ts#L209) we'd check if the section belongs to a certain list?

Reply:
It also seems to me that the purpose of groupIDs being an array is lost, the usages assume that we can only have one group id for a section. Can we not have a section that can have multiple group ids?